tests/optimizer_pytest.py

```diff
@@ -0,0 +1,24 @@
+import pandas as pd
+
+from src.backend.job import Job
+from src.backend.producer import Producer
+
+
+def test_optimize():

```

Comment:
The optimizer actually kinda works! This example might be a bit hard to visualize just from the numbers, but the optimal value is found if all jobs, excluding the last, is included (i.e. all but the last index is positive)